indexer/src/main/java/au/org/aodn/esindexer/service/GeoNetworkServiceImpl.java

```diff
@@ -22,6 +26,13 @@
 
 public class GeoNetworkServiceImpl implements GeoNetworkService {
 
+    public static final String SUGGEST_LOGOS = "suggest_logos";
+    public static final String THUMB_NAILS = "thumbnails";
+    public static final String URL = "url";
```

Comment:
These are not being used by any external classes other than this class, is there a reason it has to be `public`?

Reply:
I should have use it in test case ?